cfgov/sheerlike/templates.py

```diff
@@ -11,10 +11,9 @@ def _convert_date(date, tz):
         if type(date) in [datetime.datetime, datetime.date]:
             if isinstance(date, datetime.datetime):
                 pytzone = timezone(tz)
-                if not date.tzinfo:
-                    date = pytzone.localize(date)
-                else:
-                    date = date.astimezone(pytzone)
+                if date.tzinfo:
+                    date = date.replace(tzinfo=None)
+                date = pytzone.localize(date)
     return date
```

Comment:
I don't know if stripping the date of its timezone info is a good idea.  For example, if you started with a timezone that is in ET, you would end up with the same time but without the ET asociated with it.  Then, let's say you localize it to PT, you're *still* using the same time, but you've simply swapped the timezones from ET to PT without updating the datetime itself.